mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1080,12 +1100,40 @@ pub async fn nexus_create(
         return Ok(());
     }
 
+    let (min_cntlid, max_cntlid, resv_key) =
+        nvme_params.unwrap_or((NVME_MIN_CNTLID, NVME_MAX_CNTLID, 0x1234_5678));
+    if min_cntlid < NVME_MIN_CNTLID
```

Comment:
There is some redundancy here: we only really need: min_cntlid < NVME_MIN_CNTLID || max_cntlid < min_cntlid || NVME_MAX_CNTLID < max_cntlid

Reply:
Good spot. The upstream SPDK code is actually similarly redundant.